deploy/examples/common.yaml

```diff
@@ -671,21 +677,29 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: "psp:rook"
+  name: 'psp:rook'
 subjects:
   - kind: ServiceAccount
     name: rook-csi-rbd-provisioner-sa
     namespace: rook-ceph # namespace:operator
 ---
+# We expect most Kubernetes teams to follow the Kubernetes docs and have these PSPs.
+# * privileged (for kube-system namespace)
+# * restricted (for all logged in users)
+#
+# PSPs are applied based on the first match alphabetically. `rook-ceph-operator` comes after
+# `restricted` alphabetically, so we name this `00-rook-privileged`, so it stays somewhere
+# close to the top and so `rook-system` gets the intended PSP. This may need to be renamed in
+# environments with other `00`-prefixed PSPs.
+#
+# More on PSP ordering: https://kubernetes.io/docs/concepts/policy/pod-security-policy/#policy-order
```

Comment:
I wonder if we should remove PSPs completely since they're deprecated in K8s 1.21 and will be removed in 1.25. But we should probably save this for a v1.9 discussion...

Reply:
Agreed, can we leave a TODO or open an issue for the removal?